example_folder/example_one.rb

```diff
@@ -1 +1,3 @@
-puts("Hello Neil & Paul!")
\ No newline at end of file
+puts("Hello Neil & Paul!")
+
+puts("Neil & Paul, plese could you look at my code and verify that I haven't gone completely off track.")
```

Comment:
This is a great bit of ruby Will. You really ought to make Rails Apps for a living.